client/src/send_and_confirm_transactions_in_parallel.rs

```diff
@@ -205,8 +220,7 @@ async fn send_transaction_with_rpc_fallback(
                     // fall through on io error, we will retry the transaction
```

Comment:
nit: I can't suggest in the line above this, but you could simplify this implementation by just matching the cases in the line above, and not need `add_transaction_error_in_context`, ie:

```
ErrorKind::Io(_) | ErrorKind::Reqwest(_) |
ErrorKind::TransactionError(TransactionError::BlockhashNotFound) | 
ErrorKind::RpcError(RpcError::RpcResponseError { data: RpcResponseErrorData::SendTransactionPreflightFailure(RpcSimulateTransactionResult { err: Some(TransactionError::BlockhashNotFound, .. } ) } )
```

What do you think?

Reply:
I think you are right
let me do the changes and commit it back.